js/src/features/human-resouce/components/Note/NoteForm.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Form, Button } from "antd";
+import TextArea from "antd/lib/input/TextArea";
+import { FormErrorMap } from "MyTypes";
+import { useFormServerErrors } from "hooks/useFormServerErrors";
+
+interface Props {
+  addNote: (text: string) => any;
+  errors: FormErrorMap;
+}
+
+const TEXT_LENGTH = 256;
+
+export const NoteForm: React.FC<Props> = ({ addNote, errors }) => {
+  const [form] = Form.useForm();
+
+  useFormServerErrors(form, errors, ["text"]);
+
+  const handleFinish = (values: any) => addNote(values.text);
+
+  return (
+    <Form form={form} onFinish={handleFinish}>
+      <Form.Item
+        name="text"
+        rules={[
+          { required: true, message: "Please input message!" },
+          {
+            max: TEXT_LENGTH,
+            message: `Note text is too long. Max length is ${TEXT_LENGTH}`,
+          },
+        ]}
+      >
+        <TextArea rows={4} placeholder="Enter note text" />
```

Comment:
You can write Input.TextArea and you won’t need to import TextArea

Reply:
done